src/V3File.h

```diff
@@ -185,14 +185,31 @@ class V3OutFile VL_NOT_FINAL : public V3OutFormatter {
     // MEMBERS
     FILE* m_fp = nullptr;
 
+    char* m_bufferp = nullptr;  // Write buffer
```

Comment:
Might be cleaner to use a smart pointer here, then no need to properly free it when destroyed.

Reply:
New code doesn't use manual memory allocation